### DISCUSSION TITLE
Link not valid

### DISCUSSION THREAD

**Discussion by edge4468** [jaykali/maskphish#41] [Q&A] [answered]
![Screenshot_2021-06-02-16-11-29-893_com termux](https://user-images.githubusercontent.com/65273637/120466922-3cba2700-c3bd-11eb-91a8-eaa291e20c9d.jpg)
  

Generated link not valid due to "@" at the end of url...

Any solution?

**Comment by jaykali** [comment]
Thanks for look at the problem and report us.

We used https://is.gd website to short the urls. But is.gd server seems down. We are looking for any other alternative.

We have modified it to work without shorting URLs (Temporary solution). But we are trying hard to find a permanent solve.

**Reply by edge4468** [reply]
Great!
So is there any way to mask the link right away?

**Comment by jaykali** [comment]
Remove your maskphish folder then again git clone it. It will work now.

**Reply by edge4468** [reply]
Did it
![Screenshot_2021-06-03-10-31-53-268_com termux](https://user-images.githubusercontent.com/65273637/120589786-07faae00-c457-11eb-9387-f74318d54a55.jpg)

But this link is not valid as well maybe because there is space bewteen security and verification,I tried correcting it by clearing the space but the link became invalid again...

**Reply by jaykali** [reply]
you put the space between social engineering words. That because the error comes. Use `-`.

**Reply by jaykali** [reply]
`security-verfication`

**Reply by edge4468** [reply]
Oh thanks corrected that!
But is there any way to hide that ngrok part at the end of the link to make it look more legit?
![Screenshot_2021-06-03-12-50-55-733_com termux](https://user-images.githubusercontent.com/65273637/120604144-5bc2c280-c46a-11eb-84c1-a753354f0d2e.jpg)

**Comment by jaykali** [comment, marked as answer]
All the services is running up and working again. Fixed. Now Please clear the maskphish directory from your device and git clone it again.

**Reply by ViperDark** [reply]
![Screenshot_20211113-192424](https://user-images.githubusercontent.com/89601838/141669206-5e68a5a1-5dac-4725-adba-d48b4dcb7dac.png)
On running the masked link on chrome......any solution plsss..

**Comment by manish-frustated** [comment]
is the https://is.gd website down again
?

**Reply by jaykali** [reply]
No. You can check the website (https://is.gd/).

**Comment by joshiayush** [comment]
@jaykali https://is.gd/ website is down again. I want a way to mask my phising url right away. Is there something I can do?

**Reply by joshiayush** [reply]
@jaykali Please let me know if there is any solution as soon as you see it.

**Reply by jaykali** [reply]
That is not down. I think your ISP is blocking this service. Please use VPN or Proxychains to get it. Thanks.

**Reply by joshiayush** [reply]
@jaykali fine

**Comment by ViperDark** [comment]
![Screenshot_20211113-192424](https://user-images.githubusercontent.com/89601838/141646443-5473930d-4846-4a94-91cd-8d81bb18faa7.png)
On pasting the generated link on chrome...... @jaykali any solution please.

**Reply by jaykali** [reply]
> ![Screenshot_20211113-192424](https://user-images.githubusercontent.com/89601838/141646443-5473930d-4846-4a94-91cd-8d81bb18faa7.png) On pasting the generated link on chrome...... @jaykali any solution please.

Use VPN. I think the link was not generated properly. Your ISP is blocking something. Use a VPN before generating phishing links.

**Comment by ViperDark** [comment]
Thanks for help...it's working now.

**Comment by Muusqalalay** [comment]
Pro How to fix this problem 
https://maskphis.com-mask-phis@Error, database insert failed    plz🥲
